components/wasm-opt/src/passes.rs

```diff
@@ -163,8 +191,10 @@ pub enum Pass {
     OptimizeCasts,
     /// Optimizes instruction combinations.
     OptimizeInstructions,
-    /// Optimize Stack IR.
-    OptimizeStackIr,
+    /// Outline instructions.
+    /// TODO: This is gated behind conditional compilation in the CPP.
+    /// Need to figure out how to handle that on the rust end.
+    Outlining,
```

Comment:
As stated in the TODO, this is [gated behind conditional compilation](https://github.com/WebAssembly/binaryen/blob/353b759b230dff8fb82aeb157aeb6db360d74a49/src/passes/pass.cpp#L359-L363) in the C++ code and I'm not sure how to handle that on the Rust end. Would that just be adding a new feature and gating this enum variant behind a `cfg` attribute?